site/public/js/pdf/PDFAnnotateEmbedded.js

```diff
@@ -234,8 +234,13 @@ function render(gradeable_id, user_id, grader_id, file_name, file_path, page_num
 
                 let pdfData;
                 try {
-                    pdfData = JSON.parse(data)['data'];
-                    pdfData = atob(pdfData);
+                    pdfData = JSON.parse(data);
+                    // Checking if the response is a failure due to large file size
+                    if (pdfData.status === 'fail') {
+                        $("#pdf-error-message").text(pdfData.message).show();
```

Comment:
This should clear the tests:
```suggestion
                        $('#pdf-error-message').text(pdfData.message).show();
```